### ISSUE TITLE
at1846 registers seems to mismatch values documented elsewhere

### ISSUE THREAD

**Issue by RE-Specto** [OpenRTX/openrtx.github.io#44]
the AT1846 register map at [at1846_registers.md](https://github.com/OpenRTX/openrtx.github.io/blob/master/at1846_registers.md) appears to mismatch the [AT1846S_programming_guide.pdf](https://github.com/OpenRTX/OpenRTX-external-docs/blob/main/Datasheets%20and%20refmans/AT1846S/AT1846S_programming_guide.pdf) doc in few places like
missing 0x30[14] xtal_mode
missing 0x3A[14:12] voice_sel
mismatch values in 0x0A[5:0] pabias_voltage
missing 0x4C[15:0] instead have duplicate 0x4A
and few other inconsistencies.

I wonder which doc have the correct values? are they differs on purpose as being found experimentally to be different than documented?
@silseva any ideas?

**Comment by silseva** [comment]
Yes, the markdown page is misleading. I started writing it in an attempt of making some order between the AT1846S programming guide and the RDA1846S programming guide but I never finished it. The contents have been taken from the [RDA1846S programming guide](https://github.com/OpenRTX/OpenRTX-external-docs/blob/main/Datasheets%20and%20refmans/AT1846S/RDA1846_programming_guide.pdf), whose registers' description is consistent with what we found in the GD-77 and MD-UV380 firmware.
